acl.go

```diff
@@ -277,8 +277,10 @@ func (odbi *ovndb) rowToACL(uuid string) *ACL {
 	case string:
 		severity = cacheACL.Fields["severity"].(string)
 	case libovsdb.OvsSet:
-		for _, a := range cacheACL.Fields["meter"].(libovsdb.OvsSet).GoSet {
-			severity = a.(string)
+		if meters, ok := cacheACL.Fields["meter"]; ok {
+			for _, a := range meters.(libovsdb.OvsSet).GoSet {
+				severity = a.(string)
+			}
 		}
```

Comment:
Thanks for the PR. However, this fix doesn't make sense. The problem is apparently a copy/paste bug. The fix should replace "meter" with "severity".